### PR TITLE
Add note about newer Windows versions behavior

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8535

### CODE REVIEW

File: windows/client-management/change-default-removal-policy-external-storage-media.md [hunk 1]
```diff
@@ -45,5 +45,8 @@ To change the policy for an external storage device:
    ![In Disk Management, right-click the device and click Properties.](./images/change-def-rem-policy-1.png)
   
 6. Select **Policies**, and then select the policy you want to use.
+
+> [!NOTE]  
+> In newer Windows versions, you need to first select **Hardware**, choose the USB device under **All disk drives** and then click on **Properties**
```
Comment (illfated) [inline windows/client-management/change-default-removal-policy-external-storage-media.md:50]: cosmetic/codestyle: ending period for sentences.
```suggestion
> In newer Windows versions, you need to first select **Hardware**, choose the USB device under **All disk drives** and then click on **Properties**.
```

**Review by illfated — APPROVED** [review]
#sign-off

**Comment by illfated** [comment]
@nam31 : We may want the usual review from JohanFreelancer9, although the author review would be good too.

**Comment by nam31** [comment]
@Teresa-Motiv PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by Teresa-Motiv** [comment]
Sorry for the delay--I've been tied up with another project. This change has to be documented in our internal triage process, which I will start this morning. The text also needs to be re-edited to match current standards.

**Comment by Teresa-Motiv** [comment]
Fix has been published from the internal repo, and is now live

**Pull request closed without merging**